recipes/esmf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $(uname) == Linux ]]; then
+yum install -y gcc-gfortran
```

Comment:
Cool, good to know.

Good point on the linkage issue. I have sometimes gotten around this in the past by using `install_name_tool` or `patchelf` to [change the linkage]( https://github.com/conda/conda-recipes/blob/afa9c3ca232ab60cb0bda8bbc119b838dbbba704/openblas/build.sh#L19-L24 ) to use libgcc from conda. It is a bit hacky, but if we are using the same version it should work. We could start packaging a matching copy of gcc to solve this exact problem. Another viable solution might be to just copy the relevant compiler libraries into conda and tweak their paths using the same strategy. This could be done as part of this package or we could create a new package for this purpose only. The simplest solution may be to just statically link against the compiler libraries.